workflow/github.md

```diff
@@ -1,2 +1,138 @@
 # Working on GitHub
 
+This document outlines the rules to contribute and to create issues to a project on the organisation [Canonical-websites](https://github.com/canonical-websites/)
+
+## Table of content
```

Comment:
I wonder if all the top-level sections should be split out into their own documents? Or do we prefer longer documents with table-of-contents?

Reply:
If think it's better to have one document with all those informations (minus the labels) because it's easier to read and to manage. This document won't be read by a lot of people, unless the newcomers and the people who deliberately want to contribute. 

Let's not get lost in to many documents to read. Newcomers could be afraid by thinking there are a lot of rules to follow to contribute.

Reply:
Yep in this case I think that makes sense to me. Let's see what others think.